docs/user/user/pings/sent-by-glean.md

```diff
@@ -0,0 +1,70 @@
+# Pings sent by Glean
+
+If data collection is enabled, the Glean SDK provides a set of built-in pings that are assembled out of the box without any developer intervention.  The following is a list of these built-in pings:
+
+- [`baseline` ping](baseline.md): A small ping sent every time the application goes to foreground and background. Going to foreground also includes when the application starts.
+- [`deletion-request` ping](deletion_request.md): Sent when the user disables telemetry in order to request a deletion of their data.
+- [`events` ping](events.md): The default ping for events. Sent every time the application goes to background or a certain number of events is reached.
+- [`metrics` ping](metrics.md): The default ping for metrics. Sent approximately daily.
+
+Applications can also define and send their own [custom pings](custom.md) when the schedules of these pings is not suitable.
+
+There is also a [high-level overview](ping-schedules-and-timings.html) of how the `metrics` and `baseline` pings relate and the timings they record.
+
+### Available pings per platform
+
+| Language Binding | [`baseline` ping](baseline.md) | [`deletion-request` ping](deletion_request.md) | [`events` ping](events.md) | [`metrics` ping](metrics.md) |
```

Comment:
nit: can we remove the "ping" and just leave the name? It looks a bit weird.

<img width="761" alt="Screenshot 2021-06-04 at 15 45 06" src="https://user-images.githubusercontent.com/25176023/120811015-dadbf780-c54b-11eb-8bd0-7182aae77df2.png">